DannyBen/sla
# Add --ignore and change output format

1. Specs improvements:
   - Remove spec/cache folder
   - Test against mock server in CI as well
1. Exit with non zero when there is a failure
1. Add `--ignore "/path /another/path"` flag
1. Remove `--no-color` flag
1. Logging changes:
   - Remove `--no-log`
   - By default, we do not write to a file. Will write only if `--log FILE` is provided
   - Log file will now only contain the errors
1. Remove `$cache` global variable. Use `WebCache` singleton instead
1. Output changes:
   - No longer display the long useless tree. 
   - Instead, show a single line (that rewrites itself) with the current status, and print errors as they come.
   - When running in a non-tty environment, the lines will be printed normally (thanks to the update to Colsole 0.7.0)
